README.md

```diff
@@ -53,7 +55,7 @@ You can also use this script as a library. For instance, if you want to print ou
 ```python
 from itertools import islice
 from youtube_comment_downloader import *
-downloader = YoutubeCommentDownloader()
+downloader = YoutubeCommentDownloader(https_proxy='https://user:password@proxy.example.com:8080')
```

Comment:
I'd prefer to leave out the `https_proxy` in the example. That way the example runs out of the box.